common/domain/handler_GlobalDomainDisabled_test.go

```diff
@@ -461,6 +461,79 @@ func (s *domainHandlerGlobalDomainDisabledSuite) TestUpdateGetDomain_AllAttrSet(
 	)
 }
 
+func (s *domainHandlerGlobalDomainDisabledSuite) TestDeprecateGetDomain() {
```

Comment:
Could we do some refactoring on those tests along the way? At least for the newly added ones.
Those are already very repetitive any adding more of them makes it worse.

Reply:
What do you mean by repetitive?

Reply:
All those tests share nearly identical code. They list domain properties, register it and then assert all the properties again. I think we could make some helpers here to make it more compact and easier to read. The essence of a test gets lots in all the details around it.

Reply:
Updated the unit test.